### DISCUSSION TITLE
登录后跳转到登录前的浏览器地址

### DISCUSSION THREAD

**Discussion by ZhankaiWang** [slowlyo/owl-admin#50] [开发建议]
目前待办中无法定位到具体文档，希望能够重定向，谢谢！

**Comment by slowlyo** [comment]
退出登录不就行了

**Reply by ZhankaiWang** [reply]
希望把laravel 原有的redirector功能集成进去，譬如我要访问http://localhost/admin/#/system/admin_users/1,希望转到用户信息页面；但转到登录界面后，登录后到达不了这个地址，而是转到了系统首页

**Reply by slowlyo** [reply]
- 登录后跳转的页面, 可以在菜单管理里面控制
- 你指的应该是, 访问 `admin/#/admin_user` , 但是没有登录, 被系统重定向到了登录页, 但是你想在完成登录后, 系统继续前往 `admin/#/admin_user` 这个地址

这个会在后续版本中优化的, 属于前端的跳转逻辑, 与 laravel 无关

**Comment by ZhankaiWang** [comment]
谢谢啦
看到middleware中有这么一段代码，改了半天也没有成功，不起作用

class RedirectIfAuthenticated
{
    /**
     * Handle an incoming request.
     *
     * @param Request $request
     * @param \Closure(Request): (Response) $next
     * @param string ...$guards
     * @return Response
     */
    public function handle(Request $request, Closure $next, string ...$guards): Response
    {
        $guards = empty($guards) ? [null] : $guards;

        foreach ($guards as $guard) {
            if (Auth::guard($guard)->check()) {
                return redirect(RouteServiceProvider::HOME);
            }
        }

        return $next($request);
    }
}
